flyteadmin_config.yaml

```diff
@@ -95,29 +95,26 @@ storage:
   container: "flyte"
 queues:
   executionQueues:
-    - primary: "gpu_primary"
-      dynamic: "gpu_dynamic"
+    - dynamic: "gpu_dynamic"
       attributes:
         - gpu
-    - primary: "critical"
-      dynamic: "critical"
+    - dynamic: "critical"
       attributes:
         - critical
-    - primary: "default"
-      dynamic: "default"
+    - dynamic: "default"
       attributes:
         - defaultclusters
-    - primary: "my_queue_1"
+  workflowConfigs
```

Comment:
Is this a new change?

Reply:
workflowConfigs is not relevant anymore right?

Reply:
this file specifically was borked. we still read per-domain defaults from the config https://github.com/lyft/flyteadmin/blob/master/pkg/manager/impl/executions/queues.go#L90